cmd/janus/server.go

```diff
@@ -39,10 +53,12 @@ func RunServer(cmd *cobra.Command, args []string) {
 
 	initLogger(globalConfig.LogLevel)
 	initTracing(globalConfig.Tracing)
-	storage := initStorage(globalConfig.Storage)
-	statsClient := initStatsdClient(globalConfig.Stats)
+	storage = initStorage(globalConfig.Storage)
+	statsClient = initStatsdClient(globalConfig.Stats)
 	defer statsClient.Close()
 
+	go startPubSubLoop(storage)
```

Comment:
I would move this to another pkg, maybe `go NotificationsListener.Start()` or something wdyt?